.helm/starter/templates/extra-list.yaml

```diff
@@ -0,0 +1,8 @@
+{{- range .Values.extraDeploy }}
+---
+{{-   if typeIs "string" . }}
+{{-     tpl . $ }}
+{{-   else }}
+{{-     tpl (. | toYaml) $ }}
```

Comment:
it would be 
```
{{- tpl (. |  toYaml | nindent 0) }}
```
For line 6. The nindent counteracts the chomp at the beginning of the line to make sure that formatting is good. I've found it helpful in ensuring proper indentation. You can also use `nindent 0` so that you can actually nest your if statements inside the range block - the nindent combined with the chomp will put lines 4 and 6 in the right place

Reply:
Okay, I've adjusted the indentation and added `nindent 0` to the template